recipes/canopen/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "canopen" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/canopen-{{ version }}.tar.gz
+  sha256: 5f18651b9df6e4769b9882e969f934ae773ef24a45aabc3334b586982048d08c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.6
+    - python-can >=3.0.0
+
+test:
+  imports:
+    - canopen
+  commands:
+    - pip check
+  requires:
+    - pip
+    - msgpack-python
```

Comment:
This is no longer required once https://github.com/conda-forge/python-can-feedstock/pull/16 is merged.